Proxy/HTTP/NotesKeywordHighlighter.bambda

```diff
@@ -0,0 +1,29 @@
+/**
+ * Notes Filter Bambda
+ * This Bambda filters Proxy HTTP history for entries with notes containing a specified keyword.
+ * It checks for a specific keyword within the notes and highlights the matching entries.
+ * Users can easily modify the 'keyword' variable to suit their specific search criteria.
+ * Author: Tur24Tur
+ * GitHub: @BugBountyzip (https://github.com/BugBountyzip)
```

Comment:
Please can you change this to:
@author Tur24Tur / BugBountyzip (https://github.com/BugBountyzip)

Reply:
done thank you